site/en/docs/extensions/mv3/getstarted/tut-tabs-manager/index.md

```diff
@@ -161,14 +161,13 @@ browser.
 
 #### Request permission {: #narrow }
 
-Many features in the Tabs API do not require any permissions to use. However, in our case, we need
-to declare a permission to access two sensitive properties: the `title` and the `URL`. 
+You can use many methods in the Tabs API without requesting any permission. However, we need access to the `title` and the `URL` of the tabs; these sensitive properties require permission.
 
-We could request `"tabs"` permission, which would allow us to see the sensitive properties of all
-the browser tabs. However, to protect user privacy, we will request narrow [host
-permissions][doc-match], giving us access _only_ to the title and URL of **specific sites**.
+We could request `"tabs"` permission, but this would allow access to the sensitive properties of **all** browser tabs. Since we are only managing tabs of a particular site, we will request narrow host permissions. 
 
-Add the following code to the manifest JSON file:
+Narrow [host permissions][doc-match] allow us to protect user privacy by granting elevated permission to **specific sites**. This will grant access to the `title`, and `URL` properties, as well as additional capabilities.
```

Comment:
@dotproto Please review this copy about the tabs and narrow host permissions. Feel free to edit for clarity.

Reply:
As I was reviewing this I found myself debating whether the extension should be using the `"tabs"` permission or request host permissions for the doc site. The main issue with requesting host permissions is that we will be changing how host grants work in the foreseeable future. Today, when an extension declares an array of host permissions in the `"host_permissions"` manifest field, those permissions are granted at install time. In the future, extensions will default to running "on click" and users will be able to choose to grant an extension persistent access to a host. In other words, the current approach taken by this guide may lead to unexpected results.

IMO there are 4 approaches we could take.

1. Leave the demo and tutorial as is.
    * Pro: No extra work.
    * Con: May end up confusing and surprising developers down the road.
2. Leave the demo as is, but add a warning to the tutorial that describes how the host permissions changes will affect this approach.
    * Pro: Lets us take different approaches in different demos (not sure if that's actually relevant).
    * Con: Leaves proper handling of permissions "as an exercise for the reader" (i.e. good luck!)
3. Expand the demo and tutorial to include host permissions checks and (if necessary) a runtime host permission request.
    * Pro: More robust. Better sets expectations long-term and demonstrates how to navigate these constraints.
    * Con: Creates a decent amount of new work.
4. Switch the demo and tutorial to the tabs permission.
    * Pro: Relatively easy to implement.
    * Pro: Robust against the long term direction of the platform.
    * Con: Scary permission warning.

Short term, I think the two best options are (2) and (4). If we go with (2), we can expand to (3) further down the road.

---

I started to revise this section to dive into the tradeoffs between tabs and host permissions, but realized I was spending too much time on it and going too deep into this. In case it's useful, though, here's what I came up with.

> You can use many methods in the Tabs API without requesting any permission. However, in order to find and show a list of extensions documentation tabs, we will need to read the `title` and `url` properties of all relevant tabs.
> 
> There are two many ways we can do this: the `"tabs"` permission and host permissions. Both approaches have drawbacks.
> 
> The `"tabs"` permission grants an extension the ability to read sensitive data on **all** tabs including the title and URL of the tab. Over time, this information could be used to collect a user's browsing history. As such, if you request this permission Chrome will display a warning message at install time that says the extension can "Read your browsing history". 
> 
> Host permissions grant an extension the ability to inject scripts on a page … "Read and change your data on developer.google.com"
```

Reply:
I would go with 4. Making promises about future functionality in product documentation should be avoided. You don't know what implementation problems you'll run into that could lead to a change in design. You don't know what change in circumstances could lead to a change in priorities. The chance that we'll remember to update the article is low.

Reply:
We discussed this in today's extensions TW sync and decided to proceed with (1). When we scoped out the set of concepts we wanted to cover in these gettings started tutorials, two concepts we wanted to introduce in this example were narrow host permissions and privacy considerations. If we went with option (4), we would need to find another way to introduce these concepts. (1) also leaves the door open to update the tutorial as described in (3) once the default host permission grant change lands in Chromium.

Reply:
I'll follow the decision made at the TW sync (1). Simeon, I think your expanded copy about the differences between both choices could go under a details component. I'll play with this idea later today.

@jpmedley, you can start with the editorial review of the other files for now. Thanks!  :)

Reply:
@dotproto [Here's the preview of this section](https://deploy-preview-3328--developer-chrome-com.netlify.app/docs/extensions/mv3/getstarted/tut-tabs-manager/#narrow). I added a details component with further explanation.